.yarn/patches/app-builder-lib-npm-24.13.3-86a66c0bf3.patch

```diff
@@ -1,15 +1,82 @@
+#   than incorrectly stating that the issue was with closing the app).
+#
 # See https://github.com/laurent22/joplin/pull/11541
 diff --git a/templates/nsis/include/allowOnlyOneInstallerInstance.nsh b/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
-index fe5d45c730f36c9fe8d8cfea12e242e501b67139..af2ce5c90ac910b079e24992519bffe33d57668a 100644
+index fe5d45c730f36c9fe8d8cfea12e242e501b67139..97b27fce6798e30e3e631221435f09b3579e77c3 100644
 --- a/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
 +++ b/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
 @@ -42,7 +42,7 @@
      ${nsProcess::FindProcess} "${_FILE}" ${_ERR}
    !else
      # find process owned by current user
 -    nsExec::Exec `%SYSTEMROOT%\System32\cmd.exe /c tasklist /FI "USERNAME eq %USERNAME%" /FI "IMAGENAME eq ${_FILE}" /FO csv | %SYSTEMROOT%\System32\find.exe "${_FILE}"`
-+    nsExec::Exec `%SYSTEMROOT%\System32\cmd.exe /c tasklist /FI "USERNAME eq %USERNAME%" /FI "PID ne $pid" /FI "IMAGENAME eq ${_FILE}" /FO csv | %SYSTEMROOT%\System32\find.exe "${_FILE}"`
++    nsExec::Exec `"$SYSDIR\cmd.exe" /c tasklist /FI "USERNAME eq %USERNAME%" /FI "IMAGENAME eq ${_FILE}" /FO csv | "$SYSDIR\find.exe" "${_FILE}"`
      Pop ${_ERR}
    !endif
  !macroend
+@@ -73,7 +73,7 @@
+       !ifdef INSTALL_MODE_PER_ALL_USERS
+         nsExec::Exec `taskkill /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid"`
+       !else
+-        nsExec::Exec `%SYSTEMROOT%\System32\cmd.exe /c taskkill /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid" /fi "USERNAME eq %USERNAME%"`
++        nsExec::Exec `"$SYSDIR\cmd.exe" /c taskkill /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid" /fi "USERNAME eq %USERNAME%"`
+       !endif
+       # to ensure that files are not "in-use"
+       Sleep 300
+@@ -91,7 +91,7 @@
+           !ifdef INSTALL_MODE_PER_ALL_USERS
+             nsExec::Exec `taskkill /f /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid"`
+           !else
+-            nsExec::Exec `%SYSTEMROOT%\System32\cmd.exe /c taskkill /f /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid" /fi "USERNAME eq %USERNAME%"`
++            nsExec::Exec `"$SYSDIR\cmd.exe" /c taskkill /f /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid" /fi "USERNAME eq %USERNAME%"`
+           !endif
+           !insertmacro FIND_PROCESS "${APP_EXECUTABLE_FILENAME}" $R0
+           ${If} $R0 == 0
+diff --git a/templates/nsis/include/installUtil.nsh b/templates/nsis/include/installUtil.nsh
+index 47367741632726ba0886ac516461dbe98b7aea58..25cb4690e945ff5c77d6aa0845480a6dca318031 100644
+--- a/templates/nsis/include/installUtil.nsh
++++ b/templates/nsis/include/installUtil.nsh
+@@ -126,10 +126,11 @@ Function handleUninstallResult
+   Return
+ 
+   ${if} $R0 != 0
+-    MessageBox MB_OK|MB_ICONEXCLAMATION "$(uninstallFailed): $R0"
++    MessageBox MB_YESNO|MB_ICONEXCLAMATION "$(uninstallFailedIgnore) (error code $R0)" IDYES ignoreError
```

Comment:
The error message shown here is potentially confusing. Currently, it looks like this:
![Joplin Setup warning failed to remove old application files. Continue without removing old application files?](https://github.com/user-attachments/assets/c84f333e-68a0-439a-bed6-89220ea65dfc)

It may make sense to either:
1. Remove this message box entirely. If uninstalling the old version fails, always continue with the installation.
   - I'm not sure whether continuing with the installation in this case can cause issues.
2. Improve the warning dialog.

Reply:
As a user I wouldn't know what to answer to this. Why would I click OK or why would I Cancel, I don't have enough information to answer. 

I think we're a bit more confident than the user that proceeding with installation should be ok, so I think we can remove the dialog entirely.